app/src/main/java/campus/tech/kakao/map/data/network/api/RetrofitService.kt

```diff
@@ -0,0 +1,21 @@
+package campus.tech.kakao.map.data.network.api
+
+import campus.tech.kakao.map.BuildConfig
+import campus.tech.kakao.map.data.network.dto.SearchResponse
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Header
+import retrofit2.http.Query
+
+interface RetrofitService {
+
+    @GET("v2/local/search/keyword.json")
+    fun requestPlaces(
+        @Header("Authorization") auth: String = KEY,
+        @Query("query") query: String
+    ): Call<SearchResponse>
```

Comment:
2단계에서는 네트워크 비동기처리에도 코루틴을 활용해보시면 좋겠습니다.
```suggestion
    suspend fun requestPlaces(
        @Header("Authorization") auth: String = KEY,
        @Query("query") query: String
    ): SearchResponse
```
수정하면서 함께 생각해볼만한 내용입니다.
- 가독성 측면에서 두가지 방식의 비교
- suspend 키워드를 달았을 때 네트워크 콜이 수행되는 쓰레드
- 반환형 타입(Result<SearchResponse>, SearchResponse)에 따른 예외처리 방식

더 깊게는 suspend 키워드를 달았을 때 Retrofit 내부에서는 어떻게 enqueue가 수행되는 것인지 까지도 살펴보시면 좋겠습니다.